util/xs_scripts/docker/prepare-env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+CHECKPOINTS_DIR="checkpoints"
+RESULTS_DIR="results"
+GCPT_RESTORER="/home/zybzzz/proj/openxiangshan/tools/restorer/gcpt.fromlibcptalpha"
```

Comment:
> I think these two lines should not appear here, this way it would only run on your machine, right?

Sorry, I didn't see it earlier. Yes, but I still keep it here just to provide an example. @jensen-yan